mount/mount_windows.go

```diff
@@ -98,19 +98,32 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 		getSMBMountMutex.LockKey(source)
 		defer getSMBMountMutex.UnlockKey(source)
 
-		if output, err := newSMBMapping(allOptions[0], allOptions[1], source); err != nil {
+		var output string
+		var err error
+		username := allOptions[0]
+		password := allOptions[1]
```

Comment:
could we fail if options has less than 2 args?

Reply:
I think yes, let me work out another PR to fix it.